application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageReadService.java

```diff
@@ -25,9 +27,8 @@ public BlobStorageReadService(CloudBlobContainer cloudBlobContainer) {
 
     public void loadBlob(DocumentContentVersion documentContentVersion, OutputStream outputStream) {
         try {
-            CloudBlockBlob blob = cloudBlobContainer.getBlockBlobReference(documentContentVersion.getId().toString());
-            blob.download(outputStream);
-        } catch (URISyntaxException | StorageException e) {
+            copy(new URL(documentContentVersion.getContentUri()).openStream(), outputStream);
```

Comment:
I disagree. Why to copy directly from URL? cloudBlobContainer is configured with connection string. And we use cloudBlobContainer everywhere. We should keep it for consistency and security.

Reply:
The old way relies on the logic getBlockBlobReference(documentContentVersion.getId()

It won't be consistent for my next change that breaks that rule

Reply:
But still this is secured with Azure connection string. I think it should stay like that. Please check with Valentin.